docs/file-format.md

```diff
@@ -107,7 +107,7 @@ The `Info` column must contain the following fields:
 + `TRANSCRIPT_ID`: the transcript ID of the donor (upstream) transcript.
 + `ACCEPTER_GENE_ID`: the accepter (downstream) transcript's gene ID.
 + `ACCEPTER_TRANSCRIPT_ID`: the accepter (downstream) transcript's transcript ID.
-+ `ACCEPTER_POSITION`: the position of the break point of the ACCEPTER (downstream) transcript.
++ `ACCEPTER_POSITION`: the position of the breakpoint of the ACCEPTER (downstream) transcript.
 + `GENOMIC_POSITION`: the genomic position of the donor (upstream) transcript, in the format of `<chrom name>:<breakpoint>:<breakpoint>`.
 + `ACCEPTER_GENOMIC_POSITION`: the genomic position of the accepter (downstream) transcript, in the format of `<chrom name>:<breakpoint>-<breakpoint>`.
```

Comment:
why is one `<chrom name>:<breakpoint>:<breakpoint>` and one `<chrom name>:<breakpoint>-<breakpoint>`?

Reply:
Probably a mistake